### DISCUSSION TITLE
这周你做了什么？2022-05-13

### DISCUSSION THREAD

**Discussion by jiacai2050** [EmacsTalk/emacstalk.github.io#18] [Show and tell]
- 可以随便讨论本周你打算做什么与 Emacs 相关的事情，甚至可以寻求帮助或反馈。
- 请记住，什么都不做也没关系！
- Happy Hacking Emacs！

**Comment by jiacai2050** [comment]
- 本周给懒猫的 lsp-bridge 贡献了一小段代码，支持了 rust 语言
  - https://github.com/manateelazycat/lsp-bridge/pull/5

**Reply by loveminimal** [reply]
- 准备尝鲜一下新版本的 Emacs 
- 裁剪一下 Emacs 的配置

**Comment by zilongshanren** [comment]
录了两个emacs视频，给lsp bridge贡献了corfu orderless补全

**Comment by calsys456** [comment]
论文明天要求交稿，我却刚写到Lucid Emacs的历史😂 水过一周

好想敲代码啊a，这两天打算给cndict加点新功能，顺便玩玩posframe。能不能把所有窗口用posframe显示啊，再加个拖动啊最大最小化功能啊什么的，要它exwm作甚，一个xim死活配置不好[摔]     Windows 1和Windows 2的窗口不也不支持像素级别的移动嘛，现在emacs还有了eaf，是不是emacsOS指日可待（错乱

**Reply by jiacai2050** [reply]
感觉好混乱，年轻人不要着急，Emacs 是个可以折腾很久的东西，慢慢来

**Reply by calsys456** [reply]
> 感觉好混乱，年轻人不要着急，Emacs 是个可以折腾很久的东西，慢慢来

哈哈哈哈谢谢前辈指导！我慢慢来😁

**Comment by alaskasquirrel** [comment]
https://github.com/lisp-life/lisp-china 重新整理这个资料集

**Reply by alaskasquirrel** [reply]
第253页图灵笔记 [bilibili](https://space.bilibili.com/210738994/channel/series) 这位朋友的Emacs演示推广值得点赞。

**Reply by jiacai2050** [reply]
嗯，回来看看是否有机会找 up 主来 EmacsTalk 做一期节目

**Reply by alaskasquirrel** [reply]
可以把这位朋友的系列视频 添加入 https://github.com/EmacsTalk/emacs-for-beginners
